fgcm/fgcmStars.py

```diff
@@ -116,6 +116,8 @@ def __init__(self,fgcmConfig):
 
         self.seeingSubExposure = fgcmConfig.seeingSubExposure
 
+        self.secZenithRange = 1. / np.cos(np.radians(fgcmConfig.zenithRange))
```

Comment:
Feel free to ignore, but I've always preferred `np.deg2rad()` over `np.radians` since then in the input units are clear.

Reply:
I'll make this change in the next version of fgcm.  I don't know why I'm using `radians` here, since I do usually use `deg2rad`.